kubernetes/samples/charts/util/setup.sh

```diff
@@ -1,12 +1,36 @@
 #!/bin/bash
-# Copyright (c) 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Copyright (c) 2018,2020 Oracle Corporation and/or its affiliates.  All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
 #  This script is to create or delete Ingress controllers. We support two ingress controllers: traefik and voyager.
 
 MYDIR="$(dirname "$(readlink -f "$0")")"
 VNAME=voyager-operator  # release name of Voyager
 TNAME=traefik-operator  # release name of Traefik
+VSPACE=voyager # NameSpace for Voyager
+TSPACE=traefik   # NameSpace for Traefik
+
+helm version --short | grep v2
+[[ $? == 0 ]] && HELM_VERSION=V2
+[[ $? == 1 ]] && HELM_VERSION=V3
+
+echo "Detected Helm Version [$HELM_VERSION]"
+
+if [ "$HELM_VERSION" == "V2" ]; then
+   v_list_args=""
+   t_list_args=""
+   v_helm_delete="helm delete --purge"
+   t_helm_delete="helm delete --purge"
+   v_helm_install="helm install appscode/voyager --name $VNAME  "
+   t_helm_install="helm install stable/traefik --name $TNAME "
+else
+   v_list_args="--namespace $VSPACE "
+   t_list_args="--namespace $TSPACE "
+   v_helm_delete="helm delete --keep-history --namespace $VSPACE "
```

Comment:
why `--keep-history` and not `--purge` here?

Reply:
--purge is not supported in 3.x